### PR TITLE
New orderby filters

### PR DESCRIPTION
**Pull request by matti-lamppu**
## 🛠️ Changelog
[//]: # "Describe the changes in this pull request here."

- [x] Add new `order by` filters to application, application events, and application event schedules endpoints

   - [x] Applications
   - [x] Events
   - [x] Schedules 

- [x] Add new field filters to event reservation units for preferred order

## 🧪 Test plan
[//]: # "Help your fellow reviewer and write a short description of what's the fastest way to test your changes."

- Automated tests

## 🎫 Tickets
[//]: # "This pull request resolves all or part of the following ticket(s)"

- [TILA-2981](https://helsinkisolutionoffice.atlassian.net/browse/TILA-2981)

[TILA-2981]: https://helsinkisolutionoffice.atlassian.net/browse/TILA-2981?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: api/graphql/types/application_event_schedule/filtersets.py [hunk 1]
```diff
@@ -33,11 +34,21 @@ class ApplicationEventScheduleFilterSet(BaseModelFilterSet):
...
     text_search = django_filters.CharFilter(method="filter_text_search")
 
-    order_by = django_filters.OrderingFilter(
+    order_by = CustomOrderingFilter(
         fields=[
             "pk",
             ("application_event__id", "application_event_id"),
             ("application_event__application__id", "application_id"),
+            "applicant",
+            ("allocated_reservation_unit__unit__name_fi", "allocated_unit_name_fi"),
```
Comment (vergama) [inline api/graphql/types/application_event_schedule/filtersets.py:43]: Also `application_event_name`

Comment (matti-lamppu) [reply api/graphql/types/application_event_schedule/filtersets.py:46]: Added

**Review by vergama — COMMENTED** [review]
Looks good except that I started wondering why are the sorters for **application_event.name** `name_fi/en/sv` rather than just `name`? (Thus already before this PR, though.)
In the test env for now this GQL does work but `orderBy: "name_en"` or `orderBy: "name_sv"`, on the other hand, don't affect the sort order (but are accepted, nevertheless):
```
{ applicationEvents(applicationRound: 92, orderBy: "-name_fi") {
  edges { node {
    pk, name
  } }
} }
```

**Comment by matti-lamppu** [comment]
> Looks good except that I started wondering why are the sorters for **application_event.name** `name_fi/en/sv` rather than just `name`? (Thus already before this PR, though.) In the test env for now this GQL does work but `orderBy: "name_en"` or `orderBy: "name_sv"`, on the other hand, don't affect the sort order (but are accepted, nevertheless):
> 
> ```
> { applicationEvents(applicationRound: 92, orderBy: "-name_fi") {
>   edges { node {
>     pk, name
>   } }
> } }
> ```

@vergama For translated fields, `name` is an alias for `name_fi`, since Finnish is our default language (this is hot [django-modeltranslation](https://django-modeltranslation.readthedocs.io/en/latest/) works). So, using `name` will filter with the Finnish names, and can work that way. Still, I don't know why `name_en` and `name_sv` are accepted through the filter. I'm pretty sure it should give an error about non-existing filter. This could be a bug between `django-modeltranslation` and `django-filter`.

**Review by vergama — APPROVED** [review]

**Pull request merged**